kubetest/gke.go

```diff
@@ -657,14 +657,26 @@ func (g *gkeDeployer) getInstanceGroups() error {
 	if len(g.instanceGroups) > 0 {
 		return nil
 	}
+	igs, err := g.getInstanceGroupsFromGCloud()
+	if err != nil {
+		return err
+	}
+	return g.parseInstanceGroupsFromGCloud(igs)
+}
+
+func (g *gkeDeployer) getInstanceGroupsFromGCloud() (string, error) {
 	igs, err := exec.Command("gcloud", g.containerArgs("clusters", "describe", g.cluster,
 		"--format=value(instanceGroupUrls)",
 		"--project="+g.project,
 		g.location)...).Output()
 	if err != nil {
-		return fmt.Errorf("instance group URL fetch failed: %s", util.ExecError(err))
+		return "", fmt.Errorf("instance group URL fetch failed: %s", util.ExecError(err))
 	}
-	igURLs := strings.Split(strings.TrimSpace(string(igs)), ";")
+	return string(igs), nil
+}
+
+func (g *gkeDeployer) parseInstanceGroupsFromGCloud(igs string) error {
```

Comment:
nit: It could be more explicit and readable to return the result instead of handling assignment internally - also from tests point of view.

Reply:
I was also thinking about that, but didn't want to change previous behaviour ;) Done